types/node/fs.d.ts

```diff
@@ -821,47 +821,48 @@ declare module "fs" {
 
     export interface RmDirOptions {
         /**
+         * If an `EBUSY`, `EMFILE`, `ENFILE`, `ENOTEMPTY`, or
+         * `EPERM` error is encountered, Node.js will retry the operation with a linear
+         * backoff wait of `retryDelay` ms longer on each try. This option represents the
+         * number of retries. This option is ignored if the `recursive` option is not
+         * `true`.
+         * @default 0
+         */
+        maxRetries?: number;
+        /**
+         * @deprecated since v14.14.0 In future versions of Node.js,
```

Comment:
@Semigradsky marking this property as `@deprecated` is a problem. This makes VSCode show the entire property as deprecated, which it isn't. The behavior is changing, but it is not deprecated.

Reply:
As I see per comment using `fs.rm(path, { recursive: true, force: true })` is more preferred than `fs.rmdir(path, { recursive: true })`. `recursive: false` is default so you no need to use this property.

Reply:
Not necessarily - libraries which need to support both Node before v14 and Node after v14 need to detect the existence of `fs.rm` and fall back to `fs.rmdir`